packages/admin-app/src/components/ElectionSetupWizard/ElectionSetupWizard.tsx

```diff
@@ -20,18 +24,17 @@ export enum ElectionSetupStep {
     BasicInfo = 0,
     JointKeySelect = 1,
     JointKeyRetrieved = 2,
-    ManifestMenu = 3,
-    ManifestUpload = 4,
-    ManifestBuild = 5,
-    ManifestPreview = 6,
-    SetupComplete = 7,
+    ManifestUpload = 3,
+    ManifestPreview = 4,
+    SetupComplete = 5,
 }
 
 /**
  * Wizard to setup the election
  */
 export const ElectionSetupWizard: React.FC = () => {
     const [request, setRequest] = useState({} as SubmitElectionRequest);
+    const [manifest, setManifest] = useState({} as ValidateManifestRequest);
```

Comment:
I'm still bummed this exists. This feature should be handled by pydantic itself and never be a call. This is a hold over from previous work.

Reply:
acknowledged, I will look for ways to eliminate it in the future